### PR TITLE
Fixes for manage script

### PR DESCRIPTION
**Pull request by WarmongeR1**
Fix manage command - init
Added manage command - ru

### CODE REVIEW

File: vulyk/control.py [hunk 1]
```diff
@@ -227,4 +243,5 @@ def batch(batch_name, task_type):
         pt.add_row(values)
 
     print(pt)
+
```
Comment (mrgambal) [inline vulyk/control.py]: Why did you add two spaces above and just a single one - here?
FYI: I use those because JetBrains' IDE family do know how to fold these `region/endregion` lovely to make them look pretty.

Comment (WarmongeR1) [inline vulyk/control.py]: I use hot key `CTRL+ALT+I`  in PyCharm with default PEP8. 
In PEP8 we need to insert 2 spaces between functions and one space between class methods. Here we have many functions

**Comment by mrgambal** [comment]
:+1:
